nsv13/code/modules/overmap/weapons/damage.dm

```diff
@@ -110,6 +115,15 @@ Bullet reactions
 	var/turf/T = pick(get_area_turfs(target))
 	new /obj/effect/temp_visual/explosion_telegraph(T)
 
+/obj/structure/overmap/proc/handle_critical_failure()
+	structure_crit_no_return = TRUE //Better launch those escape pods pronto
+	priority_announce("DANGER. Ship superstructure critical failure imminent. No return threshold reached.","Automated announcement ([src])")
+	addtimer(CALLBACK(src, .proc/handle_crit_round_end, FALSE), 5 MINUTES)
+
+/obj/structure/overmap/proc/handle_crit_round_end()
+	//add a cinematic here
+	src.Destroy() //RIP
```

Comment:
Sammy J called, you're benched mate.

Reply:
OUTDATE CODE, JUST LIKE YOUR REFERENCES.